tests/ttsim/mettsim/group_by_ids.py

```diff
@@ -10,23 +10,11 @@ def sp_id(
     """
     Compute the spouse (sp) group ID for each person.
     """
-    p_id_to_sp_id: dict[int, int] = {}
-    next_sp_id: int = 0
-    result: list[int] = []
+    n = 1000000
```

Comment:
Why do we need this line? Would be good to showcase a way without any such hard-coded numbers in case people want to get inspiration from here. Same solution as in GETTSIM, I guess.